File: Module-CP/Assignments/assignment14/Pass_by_address(array)/q7.c
#include<stdio.h>
typedef struct Time{
   int hr;
   int min;
   int secs;
}Time;
void storeTime(Time*,int);
void displayTime(Time*,int);
int main(){
    Time arr[5];

    storeTime(arr,5);
    displayTime(arr,5);
}
void storeTime(Time*arr,int size){
    printf("Enter Details of Time:\n");
    for(int i=0;i<size;i++){
         printf("Enter hrs:");
        scanf("%d",&arr[i].hr);
        printf("Enter min:");
        scanf("%d",&arr[i].min);
        printf("Enter secs:");
        scanf("%d",&arr[i].secs);
    }
}
void displayTime(Time*arr,int size){
      printf("-----Date DetTime-----");
        for(int i=0;i<size;i++){
        printf("\nHrs = %d",arr[i].hr);
        printf("\nMin = %d",arr[i].min);
        printf("\nSecs = %d",arr[i].secs);
    }
}

File: Module-Core Java/Assignments/C to Java/Assignment 14/Main/q7.java
import java.util.*;
class Time{
    int hrs,min,sec;
}
public class q7{
    public static void main(String[] args) {
        Time d1 = new Time();
        Time d2 = new Time();

        Scanner sc = new Scanner(System.in);
        
        System.out.println("Hrs");
        d1.hrs=sc.nextInt();
        System.out.println("Mins");
        d1.min=sc.nextInt();
        System.out.println("Secs");
        d1.sec=sc.nextInt();
        System.out.println("Hrs");
        d2.hrs=sc.nextInt();
        System.out.println("Mins");
        d2.min=sc.nextInt();
        System.out.println("Secs");
        d2.sec=sc.nextInt();

          System.out.println("Hrs "+d1.hrs);
          System.out.println("Mins" +d1.min);
          System.out.println("Secs "+d1.sec);
          System.out.println("Hrs "+d2.hrs);
          System.out.println("Mins " +d2.min);
          System.out.println("Secs "+d2.sec);
    }
}